scala-exercises-content/src/main/scala/exercises/stdlib/Extractors.scala

```diff
@@ -10,45 +12,47 @@ import org.scalatest._
  *
  *
  * object Twice {
- *   def apply(x: Int): Int = x * 2
- *   def unapply(z: Int): Option[Int] = if (z%2 == 0) Some(z/2) else None
+ * def apply(x: Int): Int = x * 2
+ * def unapply(z: Int): Option[Int] = if (z%2 == 0) Some(z/2) else None
```

Comment:
This is the result of automatic indentation. In any case I added #60 and assigned to @fedefernandez to configure Scalariform so we can have project wide standards regarding style and indentation and they are enforced in the SBT compile phase.